src/utils.jl

```diff
@@ -155,13 +161,14 @@ function crinkle_clip(plotter::MakiePlotter{3,DH,T}, decision_fun) where {DH,T}
          plotter.dh,
          plotter.u,
          plotter.topology,
-         plotter.visible,
+         copy(plotter.visible),
```

Comment:
why the copy?

Reply:
I can make a separate PR for this bugfix if you want. The issue is that if you apply a crinkle clip and plot with the original plotter, then the visibility vector is messed up, leading to a different visualization result - but the method without "!" postfix should not modify the original object, right?

Reply:
Ah true, I guess then its fine